src/test/java/org/jenkinsci/plugins/authorizeproject/strategy/SpecificUsersAuthorizationStrategyTest.java

```diff
@@ -485,10 +486,14 @@ public void testRestInterfaceSuccess() throws Exception {
         destProject.save();
         String projectName = destProject.getFullName();
         
-        WebRequestSettings req = new WebRequestSettings(
-                wc.createCrumbedUrl(String.format("%s/config.xml", destProject.getUrl())),
+        WebRequest req = new WebRequest(
+                new URL(wc.getContextPath() + String.format("%s/config.xml", destProject.getUrl())),
                 HttpMethod.POST
         );
+        req.setAdditionalHeader(
+                j.jenkins.getCrumbIssuer().getCrumbRequestField(),
+                j.jenkins.getCrumbIssuer().getCrumb(null)
+        );

```

Comment:
Causes `java.lang.IllegalStateException: STREAMED` without this change, strangely.